modules/gapi/src/executor/gstreamingexecutor.cpp

```diff
@@ -506,6 +535,7 @@ void emitterActorThread(std::shared_ptr<cv::gimpl::GIslandEmitter> emitter,
     GAPI_ITT_STATIC_LOCAL_HANDLE(emitter_push_hndl, "emitter_push");
 
     // Now start emitting the data from the source to the pipeline.
+    GAPI_LOG_INFO(nullptr, "starting: " << emitter->name());
```

Comment:
Troubleshooting  & Logging BKMs:

Log every busy-up worker tread right before starting: to link further logger messages by thread number with actual actor by name:

```
DEBUG(:1)@timestamp1     starting: VideoEmitter  <--- mark thread :1 as VideoEmitter
DEBUG:2@timestamp2     blah blah blah
DEBUG:3@timestamp2     blah blah blah
...
DEBUG(:1)@timestamp2     blah blah blah        <---belong to VideoEmitter
```